pkg/api/v1beta1/conversion.go

```diff
@@ -707,6 +707,38 @@ func init() {
 			}
 			return nil
 		},
+
+		func(in *newer.PullPolicy, out *PullPolicy, s conversion.Scope) error {
+			switch *in {
+			case newer.PullAlways:
+				*out = PullAlways
+			case newer.PullNever:
+				*out = PullNever
+			case newer.PullIfNotPresent:
+				*out = PullIfNotPresent
+			case "":
+				*out = ""
+			default:
+				//FIXME: Should we be doing validation here?  How does this play into the UX?
```

Comment:
The alternative is just to let the invalid value through and let validation catch it.

I'm going to do that, and leave a note here try to tackle it as part of fixing user-facing error messages and versioning selectors.  OK?